mopidy_soundcloud/soundcloud.py

```diff
@@ -143,22 +174,22 @@ def send(self, request, **kwargs):
 class SoundCloudClient:
     CLIENT_ID = "93e33e327fd8a9b77becd179652272e2"
 
+    public_client_id = None
+
     def __init__(self, config):
         super().__init__()
         self.explore_songs = config["soundcloud"].get("explore_songs", 25)
-        self.http_client = get_requests_session(
-            proxy_config=config["proxy"],
-            user_agent=(
-                f"{mopidy_soundcloud.Extension.dist_name}/"
-                f"{mopidy_soundcloud.__version__}"
-            ),
-            token=config["soundcloud"]["auth_token"],
-        )
+        self.http_client = get_mopidy_requests_session(config)
         adapter = ThrottlingHttpAdapter(
             burst_length=3, burst_window=1, wait_window=10
         )
         self.http_client.mount("https://api.soundcloud.com/", adapter)
 
+        self.public_stream_client = get_mopidy_requests_session(
```

Comment:
We do want the proxy settings but I am not sure we should set our Mopidy user agent when scraping the Soundcloud website.